client/src/components/Reviews/Graph.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+class Graph extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
```

Comment:
Looks good here, you could probably condense the code down to dynamically generate each rating bar using .map if you have time.